config/locales/en.yml

```diff
@@ -25,3 +25,178 @@ en:
       %{logo_img_tag}
       <p class="lead">OnDemand provides an integrated, single access point for all of your HPC resources.</p>
     motd_title: "Message of the Day"
+    breadcrumbs:
+      home: "Home"
+      my_sessions: "My Interactive Sessions"
+      all_apps: "All Apps"
+    restart_msg_tag: |
```

Comment:
change to restart_msg_html (see https://guides.rubyonrails.org/v4.2/i18n.html#using-safe-html-translations)